projects/hr-portal/src/app/hr-portal/hr-portal.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { useState, useEffect, useCallback, useRef } from "react";
+import "./hr-portal.scss";
+
+import {
+  IgrTreeGrid,
+  IgrGridToolbar,
+  IgrGridToolbarTitle,
+  IgrGridToolbarActions,
+  IgrGridToolbarAdvancedFiltering,
+  IgrGridToolbarHiding,
+  IgrGridToolbarPinning,
+  IgrGridToolbarExporter,
+  IgrColumn,
+  IgrPaginator,
+  IgrCellTemplateContext,
+} from "igniteui-react-grids";
+import {
+  IgrIcon,
+  IgrAvatar,
+  IgrIconButton,
+  IgrButton,
+  registerIcon,
+} from "igniteui-react";
+import "igniteui-react-grids/grids/combined";
+import "igniteui-react-grids/grids/themes/light/fluent.css";
+import { dataService } from "./../services/data.service";
+import { icons } from "../data/icons/Icons";
+
+export default function HRPortal() {
+  const [data, setData] = useState<any[]>([]);
+  const [isSorted, setIsSorted] = useState(false);
+  const gridRef = useRef<IgrTreeGrid | null>(null);
+
+  useEffect(() => {
+    dataService.fetchData().then((fetchedData: any) => {
+      setData(fetchedData);
+    });
+
+    icons.forEach(
+      (element: { name: string; path: string; category: string }) => {
+        const normalizedPath = element.path.startsWith("/")
```

Comment:
Do you really need this check here?
You`ve got two cases:
1. Where you run the sample standalone, then the `import.meta.env.BASE_URL` is `/` 
2. Where you run main app, then the `import.meta.env.BASE_URL` is `/react-grid-examples/`

Reply:
it was there to ensure the absolute path, since it was needed before when I was using the import rather than the plugin. Not needed now so removed it.